### PR TITLE
Change crate's name from `Hades252` to `dusk-hades`

### PR DESCRIPTION
**Pull request by ZER0**
- Update doc tests with the new name

Resolves: #77

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
-name = "hades252"
-version = "0.12.0"
+name = "dusk-hades"
```
Comment (vlopes11) [inline Cargo.toml:2]: That is fine, but I think we need some additional mandatory entries in `Cargo.toml` to publish in crates.io

**Review by vlopes11 — APPROVED** [review]
LGTM, small observation

File: Cargo.toml (hunk 1 above)
Comment (ZER0) [reply Cargo.toml:2]: You're totally right, I always forgot them.

**Review by vlopes11 — APPROVED** [review]
LGTM

**Pull request merged**